app/components/gcse_preview_component.html.erb

```diff
@@ -0,0 +1,30 @@
+<% if course.gcse_section_complete? %>
+  <p class="govuk-body">
+    <%= required_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-body">
+    <%= pending_gcse_content(course) %>
+  </p>
+
+  <p class="govuk-body">
+    <%= gcse_equivalency_content(course) %>
+  </p>
+
+  <% if course.additional_gcse_equivalencies %>
+    <p class="govuk-body">
+      <%= course.additional_gcse_equivalencies %>
+    </p>
+  <% end %>
+<% else %>
+  <div class="govuk-inset-text app-inset-text app-inset-text--important">
+    <p class="govuk-body">
+      Please add details <%= govuk_link_to "about GCSE requirements",
+        gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
+          course.provider.provider_code,
+          course.provider.recruitment_cycle_year,
+          course.course_code,
+        ) %>.
+    </p>
+  </div>
```

Comment:
Use `govuk_inset_text` helper, and use classes to ensure left border is narrower.

```suggestion
  <%= govuk_inset_text(classes: "app-inset-text app-inset-text--narrow-border app-inset-text--important") do %>
    <p class="govuk-body">
      Please add details <%= govuk_link_to "about GCSE requirements",
        gcses_pending_or_equivalency_tests_provider_recruitment_cycle_course_path(
          course.provider.provider_code,
          course.provider.recruitment_cycle_year,
          course.course_code,
        ) %>.
    </p>
  <% end %>
```